### DISCUSSION TITLE
Output preview of the handlebar, formula, string, variable etc.

### DISCUSSION THREAD

**Discussion by nist24** [Budibase/budibase#10225] [Contributing]
Hi!

Is there a way of having an instant output preview of the handlebar, formulas, strings, variables etc.? This would help a lot.

Something like that from ToolJet:
![screenshot_tooljet](https://user-images.githubusercontent.com/77503899/230146060-0215e707-3ae1-4d0f-a8ac-85c0df2635d1.png)
When I click into a field, a small field appears with the rendered output.

Thanks and regards,

**Comment by andz-bb** [comment]
hey @nist24, thanks so much for the suggestion. We really appreciate the feedback and from personal experience I would love to see this in the product!

We have a large update to Bindings in the product roadmap and our engineering team have been doing a lot of research on what changes are needed and how to implement them. Hopefully that will include live evaluation of the output as well as many other improvements to the development experience!
